### ISSUE TITLE
ZenDesk security error related to cross-origin iframe

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#2049]
This error occurred over 1000 times, but I think it was almost all from one IP. 

We need to take a look at this and determine how serious and widespread the problem is. 

SecurityError: Failed to read the 'contentDocument' property from 'HTMLIFrameElement': Blocked a frame with orig...

https://sentry.io/mit-office-of-digital-learning/micromasters/issues/190401131/

```
SecurityError: Failed to read the 'contentDocument' property from 'HTMLIFrameElement': Blocked a frame with origin "https://micromasters.mit.edu" from accessing a cross-origin frame.
  at apply (./static/js/entry/zendesk_widget.js:194:23)
```

**Comment by noisecapella** [comment]
This usually isn't a problem because the `src` for the iframe is usually empty. The session duration of the sentry page is unusually high, maybe having the iFrame open for that long causes a problem? I think all we can do is catch and silence the error if it occurs.

**Comment by singingwolfboy** [comment]
This should be fixed by #2075.
